lib/tasks/usps_enrollment.rake

```diff
@@ -0,0 +1,71 @@
+namespace :usps_enrollment do
+
+  desc 'Pass an enrollment'
+  task pass: :environment do
+    # FIXME: Check if we're in prod, and yell at the user + fail if so!
+
+    enrollment = InPersonEnrollment.find_by_enrollment_code(ENV.fetch('ENROLLMENT_CODE', ''))
```

Comment:
I swear I have written code here previously that takes an argument, looking at `ARGV[2]` or something, but I guess it's no longer around. The preference on the project seems to be reading an environment variable.